src/eko/msbar_masses.py

```diff
@@ -350,7 +350,7 @@ def compute(
     couplings: CouplingsRef,
     order: Order,
     evmeth: CouplingEvolutionMethod,
-    matching: MatchingScales,
+    matching,
```

Comment:
```suggestion
    matching: List[float],
```
Just import `List` from the `typing` module.

Reply:
done in https://github.com/NNPDF/eko/pull/199/commits/9e8604a0ee47c701e09068db52e1eb71e2861591